wordpress-mobile/WordPress-iOS
# Plans on JP: Update UI tests for Plan Selection

Fixes #[20720](https://github.com/wordpress-mobile/WordPress-iOS/issues/20720)

Update UI tests for Plan Selection:
1. Update Domain card to Free to Paid card
2. Test Domain selection
3. Verify Plan selection appears after Domain selection

To test:

- Wait for UI to complete

PR submission checklist:


https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/8dbe659b-bab0-48fe-bed9-2ff3e32f8231
